.pipelines/deployment/ServiceGroupRoot/Scripts/pushAgentToAcr.sh

```diff
@@ -74,6 +74,13 @@ TOKEN=$(echo $LOGIN_INFO | jq -r '.accessToken')
 SOURCE_IMAGE_FULL_PATH=${MCR_REGISTRY}${DEV_MCR_REPOSITORY}:${IMAGE_TAG}
 AGENT_IMAGE_FULL_PATH=${PROD_ACR_REPOSITORY_WITHOUT_SLASH}:${IMAGE_TAG}
 
+# Install oras
```

Comment:
this might not needed if we use right image for shell extension:

Reply:
Thanks @wanlonghenry ! Updated the PR.